django-server/server/server/urls.py

```diff
@@ -16,7 +16,9 @@
 from django.contrib import admin
 from django.urls import include, path
 
+
 urlpatterns = [
     path('', include('home.urls')),
     path('admin/', admin.site.urls),
+    path('api/', include('api.urls')),
```

Comment:
Can we use `api/beta/` in here? It's a good practice to have API version in URL.

Reply:
good idea, changed